pipeline/ocr/postprocess.py

```
@@ -57,8 +57,14 @@ def parse_unified_diff(text: str) -> dict[str, set[int]]:
             continue
         if line.startswith("@@"):
             m = HUNK_RE.match(line)
-            if not m or path is None:
+            if not m:
                 raise ValueError(f"不可解析的 hunk 头: {line!r}")
+            if path is None:
+                # 删除文件（+++ /dev/null）后的 hunk 头：b 侧不存在——跳过
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)